static/app/views/replays/deadRageClick/constructSelector.tsx

```diff
@@ -52,5 +53,21 @@ export default function constructSelector(element: ReplayClickElement) {
     alt +
     title;
 
-  return {fullSelector, selector};
+  const nativeSelector = tag + id + classes + role + ariaLabel + testId + alt + title;
+
+  const querySelector =
+    tag +
+    id +
+    (role ? fullRole : '') +
```

Comment:
if i can read my own code, i _think_ if `trimAttribute` does this job for us, no need to manually return `''`

Reply:
i could be wrong though 😶‍🌫️